config/locales/devise/en.yml

```diff
@@ -5,22 +5,22 @@ en:
       already_confirmed: Your email address has already been confirmed. %{action}
       confirmed: You have confirmed your email address
       confirmed_but_must_set_password: You have confirmed your email address
-      send_instructions: >
-        You will receive an email with instructions for how to confirm your
-        email address in a few minutes.
-      send_paranoid_instructions: >
-        You will receive an email with instructions for how to confirm your email
-        address in a few minutes.
+      send_instructions: 'You will receive an email with instructions for how to confirm your email address in a few minutes.
+
+'
```

Comment:
Ok so I have a compromise!

It turns out naive `YAML.dump` on my machine outputs wraps lines at around 100c by default, without quotes if it can.

I made a script that strips whitespace on YAML files in-place and dumps them back, which has the default line wrapping of 100c.

Then I added a lint spec that makes sure the files are formatted to match this, so hopefully this can help keep our files consistent moving forward?